pkg/apis/componentconfig/types.go

```diff
@@ -415,6 +415,9 @@ type KubeletConfiguration struct {
 	// iptablesDropBit is the bit of the iptables fwmark space to use for dropping packets. Kubelet will ensure iptables mark and drop rules.
 	// Values must be within the range [0, 31]. Must be different from IPTablesMasqueradeBit
 	IPTablesDropBit int32 `json:"iptablesDropBit"`
+	// experimentalHostUserNamespaceDefaulting is an experimental flag to enable defaulting of the UsernsMode for
+	// a container.
```

Comment:
Comment about this requiring remapping to be enabled in the daemon (or, ahem, container runtime?)